flowetl/etl/etl/etl_utils.py

```diff
@@ -114,7 +114,13 @@ def construct_etl_dag(
         dag_id=f"etl_{cdr_type}",
         schedule_interval=None,
         default_args=default_args,
-        template_searchpath=config_path, # template paths will be relative to this
+        template_searchpath=config_path,  # template paths will be relative to this
+        user_defined_macros={
+            "extract_table": lambda execution_time: f"etl.x{ cdr_type }_{ execution_time }",
```

Comment:
I think I'd prefer if we had a verb in the lambdas, e.g. `get_extract_table`, `get_transform_table`, `get_load_table` (or similar). This makes the code where they are being used slightly more intuitive to read and understand imho.

Also, can we rename the argument from `execution_time` to `execution_date` to be consistent with both Airflow's terminology and the value that's actually parsed to them?